### PR TITLE
NiFi Quickstart

### PR DESCRIPTION
**Pull request by aheyne**
Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-quickstart-nifi/README.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  web site for a download.)
+
+
+
+
+
+
+
+
+
```
Comment (jnh5y) [inline geomesa-quickstart-nifi/README.md]: Lots of whitespaces?  Maybe clean'em up?

File: assets/geomesa-quickstart-nifi/GeoMesa_Quickstart.xml [hunk 2]
```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" ?>
+<template encoding-version="1.0">
```
Comment (jnh5y) [inline assets/geomesa-quickstart-nifi/GeoMesa_Quickstart.xml:2]: What is this file used for?  I don't see it referenced below?

Comment (aheyne) [reply assets/geomesa-quickstart-nifi/GeoMesa_Quickstart.xml:2]: Good catch, I had a broken link in the docs below. This is the NiFi flow template. Once they have the processors installed they can upload this file and it wires up everything for them so they can just hit play.

File: geomesa-quickstart-nifi/README.md (hunk 1 above)
Comment (aheyne) [reply geomesa-quickstart-nifi/README.md]: What do you have against whitespaces? 

Fixed

Comment (jnh5y) [reply geomesa-quickstart-nifi/README.md]: Nothing 


Just trying to save (white)spaces.

File: geomesa-quickstart-nifi/README.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+```bash
+$ mvn clean install -am -pl :geomesa-accumulo-dist_2.11
+```
+
+Upload the NiFi flow template
+-----------------------------
+
+The NiFi template provided [here](../assets/geomesa-quickstart-nifi/GeoMesa_Quickstart.xml)
+sets up a basic NiFi flow from a GetFile processor to a PutGeoMesa processor.
```
Comment (jnh5y) [inline geomesa-quickstart-nifi/README.md:107]: How does this file 'set up' a NiFi flow?  

Do we upload it?  If so, maybe a word or two would help.

Comment (aheyne) [reply geomesa-quickstart-nifi/README.md:107]: FYI, it's a template that you upload with an import template option. Then it's populated in a templates _thing_ that you can import like a processor.

**Pull request merged**